hardware/asic/skywater/config.tcl

```diff
@@ -1,36 +1,32 @@
 # User config
-
 set ::env(DESIGN_NAME) system
-
 # Change if needed
 set ::env(VERILOG_FILES) "\
```

Comment:
do a get ::env of VSr first and yo don't need to list manually

Reply:
Will VSRC give me all the paths for source file from inside the Openlane/designs/src. 
This statement just tells OpenLane that these are the verilog source files. These all paths for all source files have to be " designs/my _design/src " . VSRC have all the paths from iob-soc which OpenLane does not recognize. 
Still I can do  designs/src/*.v to tell the OpenLane about all the source files and it will not be hardwired but as I know there are files like gen_if.v which is actually a header file and if i do a *.v then OpenLane treats gen_if.v as verilog source file and throws error. So these are the reasons for manual list

Reply:
what we can do is to make a list of all the files that are there in src folder in OpenLane/designs/my_design and pass it to config.tcl
This list will be updated when vsrc will change and there will be no manual list
this is my suggestion
regards